src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,207 @@
+package duke;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * Storage deals with the file that stores the items in the user's local memory so that
+ * the user can retrieve his tasks when he loads up Duke in the future after exiting.
+ */
+public class Storage {
+    private File file;
+
+    public Storage(String filePath) {
+        this.file = new File(filePath);
+    }
+
+    public File load() {
+        return file;
+    }
+
+    /**
+     * Changes a single task in the tasklist to be in a 'done' status.
+     *
+     * @param description Description of the item to be found in the taskList.
+     * @param cl Class of the item that will be found in the taskList.
+     * @param dukeData The file that is to be deleted so this new file will be written.
+     * @return The new file that has taken the place of the old file.
+     * @throws IOException To handle the exception where file is not read.
+     */
+    public File doneChanger(String description, String cl, File dukeData) throws IOException {
+        Scanner sc = new Scanner(dukeData);
+        int count = 0;
+        boolean changed = false;
```

Comment:
Maybe a prefix can be added to the variable name ?
```suggestion
        boolean hasChanged = false;
```